cmd/kwil-cli/cmds/database/call.go

```diff
@@ -44,25 +43,20 @@ kwil-cli database call get_user --dbid 0x9228624C3185FCBcf24c1c9dB76D8Bef5f5DAd6
 )
 
 func callCmd() *cobra.Command {
-	var gwAuth, logs, signCall bool
+	var gwAuth, logs bool
 
 	cmd := &cobra.Command{
 		Use:     "call <procedure_or_action> <parameter_1:value_1> <parameter_2:value_2> ...",
 		Short:   "Call a 'view' action, returning the result.",
 		Long:    callLong,
 		Example: callExample,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// We only need a private key if using gateway auth or signing the
-			// call message.
-			dialFlags := common.WithoutPrivateKey
+			// AuthenticatedCalls specifies that the call should be authenticated with the private key
+			// if the call is made to the Kwild node with private mode enabled. Else, no authentication is required.
+			dialFlags := common.AuthenticatedCalls
 			if gwAuth {
-				// overwrite the WithoutPrivateKey flag, and add the UsingGateway flag
+				// If the call is made to a gateway, the call should be authenticated with the private key.
 				dialFlags = common.UsingGateway
```

Comment:
I think we should rename `AuthenticatedCalls`.  Now it indicates that the following context: (a) using `DialClient` for a call, AND (b) NOT using gateway.  So, it's like... `NonGatewayCall`?  :laughing: